extensions/rich_text_components/components_test.py

```diff
@@ -110,6 +110,9 @@ def test_link_validation(self):
         }, {
             'url-with-value': 'http://link.com',
             'text-with-value': 1234
+        }, {
```

Comment:
I just checked this file has some issues:

- The docstrings for all functions are incorrect.
- We do not check specific Exception messages.

Would it be possible for you to fix this in the PR. If not, can you please create an issue to keep track.